### PR TITLE
Initial stab at full test

### PR DESCRIPTION
**Pull request by odcinek**
This is quite brittle, but will do for starters

### CODE REVIEW

File: test/run.sh [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+export HOME='/tmp/gd'
+export REPO='/tmp/gd/repo1'
+mkdir -p $REPO
+git config --global user.email "foo@bar.com"
+git config --global user.name "Foo Bar"
+(mkdir $REPO/recipes && mkdir $REPO/roles && echo 'name "foo"' > $REPO/roles/foo.rb && echo 'name "foo"' > $REPO/metadata.rb && git init $REPO/ && cd $REPO && git add * && git commit -m'foo') &>/dev/null
+bundle exec chef-zero &
+ZERO=$!
+sleep 5
+bundle exec grocery-delivery -n -c ./test/gd.rb
```
Comment (jaymzh) [inline test/run.sh]: why start up chef-zero if you're running in dryrun mode? or rather, why run in dryrun mode if you're starting chef-zero?

**Comment by facebook-github-bot-0** [comment]
Thank you for reporting this issue and appreciate your patience. We've  notified the core team for an update on this issue. We're looking for a response within the next 30 days or the issue may be closed.

**Comment by jaymzh** [comment]
ping before bot reaps.

**Comment by odcinek** [comment]
Better version in https://github.com/facebook/grocery-delivery/pull/20

**Pull request closed without merging**